src/business.ts

```diff
@@ -232,7 +232,7 @@ export class BusinessApi {
    */
   async transition(status: Partial<Transition>): Promise<{
     success: boolean,
-    body?: Transition,
+    business?: Transition,
```

Comment:
Is this a `business` or a `transition`, or since it's the transition of a state on the Business, is it a `business`? I'm just wondering what you, as a user of this - not the author, would want to make it clear...